sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
-        })
-        return individual;
-    }).catch(err => console.error(err));
-}
+  return models.Individuals.findOne({
+    where: {
+      IndividualID: IndividualID
+    }
+  })
+    .then(individual => {
+      individual.update({
+        IndividualName: newName,
+        OriginalPostion: newPosition, //TODO: Update this.
+        LinkedInUrl: newUrl,
+        Comments: newComments
+      });
+      return individual;
+    })
+    .catch(err => console.error(err));
+};
 
-deleteIndividual = (IndividualID) => {
-    // I can't get the delete for an individual to in turn
-    //   delete rows from EmployeeHistory, so I'm doing 
-    //   this the long way until I can ask about Sequelize
-    //   Cascade. --Sean
-    //
-    // Modfying this to avoid a race condition -- Aaron
-    return new Promise((resolve,reject)=>{
-        models.EmployeeHistory.destroy({
-            where: {
-                IndividualID: IndividualID
-            }
-        }).then((deletedHistory) => {
-            models.Individuals.destroy({
-                where: {
-                    IndividualID: IndividualID
-                }
-            }).then((deletedIndividual) =>{
-                console.log("deletion resolved")
-                resolve(deletedIndividual);
-            })
-        }).catch(err => {console.error(err)
-                        reject(err)
-        })
+deleteIndividual = IndividualID => {
+  // I can't get the delete for an individual to in turn
+  //   delete rows from EmployeeHistory, so I'm doing
+  //   this the long way until I can ask about Sequelize
+  //   Cascade. --Sean
+  //
+  // Modfying this to avoid a race condition -- Aaron
+  return new Promise((resolve, reject) => {
+    models.EmployeeHistory.destroy({
+      where: {
+        IndividualID: IndividualID
+      }
     })
-}
+      .then(deletedHistory => {
+        models.Individuals.destroy({
+          where: {
+            IndividualID: IndividualID
+          }
+        }).then(deletedIndividual => {
+          console.log("deletion resolved");
+          resolve(deletedIndividual);
+        });
+      })
+      .catch(err => {
+        console.error(err);
+        reject(err);
+      });
+  });
+};
 
 //---------------Modify Company---------------//
 modifyCompany = (CompanyID, alteredCompanyName) => {
-    return models.Companies.findOne({
-        where: {
-            CompanyID: CompanyID
-        }
-    }).then(company => {
-        company.update({
-            CompanyName: alteredCompanyName
-        })
-        return company;
-    }).catch(err => console.error(err));
-}
+  return models.Companies.findOne({
+    where: {
+      CompanyID: CompanyID
+    }
+  })
+    .then(company => {
+      company.update({
+        CompanyName: alteredCompanyName
+      });
+      return company;
+    })
+    .catch(err => console.error(err));
+};
 
 //---------------Modify Fund---------------//
 modifyFund = (FundID, alteredFundName) => {
```

Comment:
The git difference here looks like it's a result of git thinking two nearly identical functions were derived from the same source. It's hard to tell since everything was reformatted.